sample-geovelocity/siddhi-execution-geovelocity/src/main/siddhi-files/IS_ANALYTICS_GEOVELOCITY.siddhi

```diff
@@ -0,0 +1,174 @@
+@App:name("IS_ANALYTICS_GEOVELOCITY")
+@App:description("Description of the plan")
+
+-- Input stream with the Geovelocity data of usernaem,login ip and login time.
+@Source(type = 'http-request', source.id='testsource', receiver.url="https://localhost:8280/IS_ANALYTICS_GEOVELOCITY/InputStream",@map(type='json', @attributes(messageId='trp:messageId', username='$.event.username', loginTime='$.event.loginTime', loginIp = '$.event.loginIp')))
+define stream InputStream (
+    messageId string,
+    username string,
+    loginIp string,
+	loginTime long);
```

Comment:
not properly formatted.

Reply:
Fixed in 16a9ffcc4a35e7a36d08a524d44fba74b655e12a